DataFormats/Common/src/classes.h

```diff
@@ -1,34 +1,118 @@
+#include "DataFormats/Common/interface/OneToOne.h"
+#include "DataFormats/Common/interface/OneToValue.h"
+#include "DataFormats/Common/interface/OrphanHandleBase.h"
+#include "DataFormats/Common/interface/OrphanHandle.h"
+#include "DataFormats/Common/interface/OwnArray.h"
 #include "DataFormats/Common/interface/OwnVector.h"
+#include "DataFormats/Common/interface/PathStatus.h"
+#include "DataFormats/Common/interface/ProductData.h"
+#include "DataFormats/Common/interface/Ptr.h"
+#include "DataFormats/Common/interface/PtrVectorBase.h"
+#include "DataFormats/Common/interface/PtrVector.h"
 #include "DataFormats/Common/interface/RangeMap.h"
+#include "DataFormats/Common/interface/RefCoreGet.h"
+#include "DataFormats/Common/interface/RefCore.h"
+#include "DataFormats/Common/interface/refcore_implementation.h"
+#include "DataFormats/Common/interface/RefCoreStreamer.h"
+#include "DataFormats/Common/interface/RefCoreWithIndex.h"
+#include "DataFormats/Common/interface/Ref.h"
+#include "DataFormats/Common/interface/RefHolderBase.h"
+#include "DataFormats/Common/interface/RefHolder.h"
+#include "DataFormats/Common/interface/RefHolder_.h"
+#include "DataFormats/Common/interface/RefItemGet.h"
+#include "DataFormats/Common/interface/RefKeyTrait.h"
+#include "DataFormats/Common/interface/RefProd.h"
+#include "DataFormats/Common/interface/RefToBase.h"
+#include "DataFormats/Common/interface/RefToBaseProd.h"
 #include "DataFormats/Common/interface/RefToBaseVector.h"
-#include "DataFormats/Common/interface/VectorHolder.h"
+#include "DataFormats/Common/interface/RefToPtr.h"
+#include "DataFormats/Common/interface/RefTraits.h"
 #include "DataFormats/Common/interface/RefVectorBase.h"
+#include "DataFormats/Common/interface/RefVector.h"
+#include "DataFormats/Common/interface/RefVectorHolderBase.h"
+#include "DataFormats/Common/interface/RefVectorHolder.h"
+#include "DataFormats/Common/interface/RefVectorIterator.h"
+#include "DataFormats/Common/interface/RefVectorTraits.h"
+#include "DataFormats/Common/interface/SecondaryEventIDAndFileInfo.h"
+#include "DataFormats/Common/interface/SortedCollection.h"
+#include "DataFormats/Common/interface/TestHandle.h"
+#include "DataFormats/Common/interface/ThinnedAssociation.h"
+#include "DataFormats/Common/interface/traits.h"
+#include "DataFormats/Common/interface/TransientDataFrame.h"
+#include "DataFormats/Common/interface/Trie.h"
 #include "DataFormats/Common/interface/TriggerResults.h"
-#include "DataFormats/Common/interface/Wrapper.h"
-#include "DataFormats/Common/interface/FillView.h"
-#include "DataFormats/Common/interface/DataFrame.h"
-#include "DataFormats/Common/interface/DataFrameContainer.h"
-#include "DataFormats/Common/interface/DetSetVectorNew.h"
-#include "DataFormats/Common/interface/ConstPtrCache.h"
-#include "DataFormats/Common/interface/BoolCache.h"
-#include "DataFormats/Common/interface/PtrVectorBase.h"
+#include "DataFormats/Common/interface/ValidHandle.h"
```

Comment:
There is absolutely no reason to make a ROOT dictionary for ValidHandle. If `classes.h` is intended just for dictionary generation, putting all headers into the file doesn't seem like a good idea. If the intent is preparation for C++ modules, then, OK, I guess.

Reply:
I see. It is a preparation for C++ modules, but why ValidHandle.h is in interface/ if it is not supposed to be exposed to users? (Shouldn't it be under src if it was like an internal header?)

Reply:
Up until now, we've only used classes.h to create the ROOT dictionaries we need for either I/O or for access to member functions of certain classes. `ValidHandle` is not necessary for either of those cases but it is needed for other code in CMSSW.

If classes.h is now being used as preparation for C++ modules then `ValidHandle` should be there.